fastlane/lib/fastlane/setup/setup.rb

```diff
@@ -244,7 +244,7 @@ def setup_gemfile!
 
     def ensure_gemfile_valid!(update_gemfile_if_needed: false)
       gemfile_content = File.read(gemfile_path)
-      unless gemfile_content.include?("https://rubygems.org")
+      unless gemfile_content.lines.any? { |line| URI(line.strip).host == "rubygems.org" rescue false }
```

Comment:
I doubt this will work since the line is not URL `source 'https://rubygems.org'` or `source "https://rubygems.org"`, plus the case difference.